app/controllers/bulk_tagging/collections_controller.rb

```diff
@@ -0,0 +1,15 @@
+module BulkTagging
```

Comment:
So originally it was like that, but it makes the routes more complicated and less readable (custom get/post entries with custom paths etc).

By using the conventional way in rails of having a resource and a separate controller we make the routes more readable (at the expense of having more controller classes). But that could be good, otherwise the original controller will be quite big after we are done with the bulk tagging.

What do you think?